src/components/Tool/FixMailingAddresses/FixMailingAddresses.test.tsx

```diff
@@ -108,13 +108,24 @@ describe('FixMailingAddresses', () => {
     ).not.toBeInTheDocument();
   });
 
-  it('should count total contacts', async () => {
+  it('should count total contacts and startDate', async () => {
```

Comment:
Can you also add a quick test case where it uses `createdAt`? That line isn't being covered in tests.